### PR TITLE
[FIX] Journal migration encoding

### PR DESCRIPTION
**Pull request by hschink**

### CODE REVIEW

**Comment by hschink** [comment]
@meeee can you have a look on this? Thanks in advance! :cherry_blossom:

File: db/migrate/migration_utils/legacy_journal_migrator.rb [hunk 1]
```diff
@@ -278,7 +278,14 @@ def update_data_journal(id, to_insert)
     def deserialize_changed_data(journal)
       changed_data = journal["changed_data"]
       return Hash.new if changed_data.nil?
-      YAML.load(changed_data)
+
+      current_yamler = YAML::ENGINE.yamler
```
Comment (mfrister) [inline db/migrate/migration_utils/legacy_journal_migrator.rb:282]: A comment on why we need the old yaml engine would be good (other string serialization format, which uses two backslashes, psych doesn't understand and results in broken special characters when reading with psych).

**Comment by mfrister** [comment]
Apart from the one comment I'd like, this looks good :+1:

**Comment by hschink** [comment]
@meeee thanks for the hint! I hope the comment is reasonable.

**Pull request merged**